cmd/curio/internal/translations/locales/zh/messages.gotext.json

```diff
@@ -0,0 +1,621 @@
+{
+    "language": "zh",
+    "messages": [
+        {
+            "id": "This interactive tool will walk you through migration of Curio.\nPress Ctrl+C to exit at any time.",
+            "message": "This interactive tool will walk you through migration of Curio.\nPress Ctrl+C to exit at any time.",
+            "translation": "此互动工具将引导您完成Curio的迁移。\n随时按Ctrl+C退出。"
+        },
+        {
+            "id": "This tool confirms each action it does.",
+            "message": "This tool confirms each action it does.",
+            "translation": "此工具确认其执行的每个操作。"
+        },
+        {
+            "id": "Ctrl+C pressed in Terminal", 
+            "message": "Ctrl+C pressed in Terminal",
+            "translation": "在终端中按下Ctrl+C"
+        },
+        {
+            "id": "Verifying Sectors exist in Yugabyte.",
+            "message": "Verifying Sectors exist in Yugabyte.",
+            "translation": "正在验证Yugabyte中的扇区是否存在。"
+        },
+        {
+            "id": "Error verifying sectors: {Error}",
+            "message": "Error verifying sectors: {Error}",
+            "translation": "验证扇区时出错：{Error}",
+            "placeholders": [
+                {
+                    "id": "Error",
+                    "string": "%[1]s",
+                    "type": "string",
+                    "underlyingType": "string",
+                    "argNum": 1,
+                    "expr": "err.Error()"
+                }
+            ]
+        },
+        {
+            "id": "Sectors verified. {I} sectors found.",
+            "message": "Sectors verified. {I} sectors found.",
+            "translation": "已验证扇区。找到了{I}个扇区。",
+            "placeholders": [
+                {
+                    "id": "I",
+                    "string": "%[1]d",
+                    "type": "[]int",
+                    "underlyingType": "[]int",
+                    "argNum": 1,
+                    "expr": "i"
+                }
+            ]
+        },
+        {
+            "id": "Never remove the database info from the config.toml for lotus-miner as it avoids double PoSt.",
+            "message": "Never remove the database info from the config.toml for lotus-miner as it avoids double PoSt.",
+            "translation": "从config.toml中永远不要删除lotus-miner的数据库信息，因为它避免了双PoSt。"
```

Comment:
```suggestion
            "translation": "从config.toml中永远不要删除lotus-miner的数据库信息，因为它避免重复PoSt。"
```

Reply:
Thank you, but I realize I need to change that line since it's not the approach we are going with now. 
It needs to say:
"Do not run lotus-miner while running curio (with PoSt enabled) as it risks double-post."
Can you translate that for me?